fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraBinaryImpl.java

```diff
@@ -445,6 +448,44 @@ private static void decorateContentNode(final Node contentNode, final Collection
     }
 
     @Override
+    public FedoraResource getTimeMap() {
+        try {
+            final Node container = getContainingNode(getNode()).get();
```

Comment:
@lsitu Is this implementation assuming that the BinaryServiceImpl will be refactored to use a new nt:folder node as an enclosing node for the binary nt:file node.

https://github.com/lsitu/fcrepo4/blob/60d6c54bcb8ed28563fe9d8ad86814b6c69a4dc0/fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/services/BinaryServiceImpl.java#L57

Reply:
@mohideen No. I don't think so. This is retrieve the TimeMap for binary.